Patch 99_100c - add class_attrib_id to phenotype -e100

ima23 opened the pull request.

helensch commented on sql/patch_99_100_c.sql:
```diff
@@ -0,0 +1,21 @@
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+# add attrib_type_id column to phenotype table
```
column being added is attrib_id not attrib_type_id

ima23 replied:
thanks, well spotted!

ima23 commented:
Thanks for the checks @helensch. I've fixed the comments and added the foreign keys file entry. 
I have decided to rename the new column to `type_attrib_id` to be similar to the `class_attrib_id` column name for variation table.

helensch commented:
Thanks for the updates. The column name type_attrib_id may be confused with attrib_type_id used in tables attrib, attrib_type, phenotype_feature_attrib, protein_function_predictions_attrib

sarahhunt commented on sql/table.sql:
```diff
@@ -406,6 +406,7 @@ CREATE TABLE IF NOT EXISTS `phenotype_feature_attrib` (
 @column name                 Phenotype short name. e.g. "CAD".
 @column description varchar  Phenotype long name. e.g. "Coronary Artery Disease".
+@column class_attrib_id      Phenotype type attribute id. e.g. for attribute non_specified
 
```
This could be more specific, less attributey, for example - 'Class of phenotype entry, eg trait, unspecified, tumour - used for filtering'

ima23 replied:
Thanks @sarahhunt, yes, indeed. Have updated it in the next commit.

ima23 commented:
also needed :PR:  https://github.com/Ensembl/ensembl-variation/pull/508
